### ISSUE TITLE
VS2015 / Resharper 9.1 EAP02 support

### ISSUE THREAD

**Issue by BradBarnich** [xunit/resharper-xunit#32]
I need the EAP to run MSTest tests, but also have projects using xunit. 

Simply expanding the Wave version on the nuspec allowed me to install the plugin. The icons show up on a test but running a test just spins. 

I have some time to get this working, before I dig in, does 9.1 support require a new version of the SDK, and is there an EAP of the SDK available?

**Comment by citizenmatt** [comment]
Yes, it requires a new version of the SDK, which isn't technically available yet (although you should mostly be able to use the `.nupkg` files in the `%LOCALAPPDATA%\JetBrains\installations` folder). Also, EAP03 has just been uploaded.

With the spinning icons issue - is that for all test projects? Does it happen for the same projects in 9.0? There's another issue (#25) where this is being reported, but I just haven't seen it in action.

**Comment by radenkozec** [comment]
Hi there. I am not able to run xUnit tests with Resharper 9.0 Update1 and XUnit.net 2.0.4 for resharper.
I am on Visual Studio 2015 CTP6.
Do I need to use some EAP of Resharper to be able to run tests under ASP.NET vNext?

**Comment by citizenmatt** [comment]
Are you using 9.0 update 1, or the 9.1 EAP? This issue was discussing 9.1 EAP, which isn't currently supported. Also, what version of xunit are you using? 2.0.4 requires RC4.

**Comment by radenkozec** [comment]
I am using 9.0 update 1.
I am using  "xunit": "2.0.0-rc4-build2924"

I did not wanted to open new issue. 

I cannot get it run from Resharper 9.0 update 1.
Do I need to use some EAP? 
Do you support Visual Studio 2015 xUnit tests?

**Comment by citizenmatt** [comment]
Oh, never worry about starting a new issue - they're cheap, and easy to create/close :)

I've just noticed that you're talking about asp.net. This isn't supported right now. ReSharper itself doesn't support it - it's a new execution environment, which means ReSharper needs to launch a new hosting executable to run the tests, and then I'll be able to port the xunit runner to work with it. I've added issue #38 to track this.
